### ISSUE TITLE
Emote ''FIX''

### ISSUE THREAD

**Issue by snattor** [night/BetterTTV#1413]
So a while back you decided to ''fix'' some of the emotes that had ''bad'' cropping and there are a couple of us that really miss how they used to look, especially the original emote ''JKanStyle''. Apparently FFZ has an option to undo this ''fix'' but I dont see any change. So if you could add an option to undo this ''fix'' I would be very grateful.
much love

**Comment by snattor** [comment]
Not even giving us an option?

**Comment by night** [comment]
No. We will not add an option for every feature we add, especially for improvement features that don't need to be toggled like this one. That doesn't make sense and doesn't scale.

**Comment by snattor** [comment]
Just saying there are people that don't think that this feature was an improvement. well rip JKanStyle

**Comment by snattor** [comment]
https://gyazo.com/8696c9f066f1aa1d850a8241772b2964
